core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -249,8 +248,7 @@ public InteractiveTransactionResponse deposit(
             .transactionId(txnId)
             .status(INCOMPLETE.toString())
             .kind(Sep24Transaction.Kind.DEPOSIT.toString())
-            .amountIn(strAmount)
-            .amountOut(strAmount)
+            .requestedAmount(strAmount)
```

Comment:
Why are we removing amount_in and amount_out here? I think we may want to hold off on adding these requestedAmount changes until we understand the use case.

Reply:
Because we don't know amount_in and amount_out during the request time, they should be set by the business server